osu.Game.Rulesets.Osu/Edit/OsuSelectionHandler.cs

```diff
@@ -84,15 +85,18 @@ public override bool HandleReverse()
             return true;
         }
 
-        public override bool HandleFlip(Direction direction)
+        public override bool HandleFlip(Direction direction, bool flipOverOrigin)
         {
             var hitObjects = selectedMovableObjects;
 
-            var selectedObjectsQuad = getSurroundingQuad(hitObjects);
+            if (hitObjects.Length == 1 && !flipOverOrigin)
+                return false;
```

Comment:
This check makes it so that you can't flip a slider over itself even though the buttons are there:

https://user-images.githubusercontent.com/20418176/148277458-8c80a240-87a1-45d7-8211-dcdf18403ca0.mp4

Was that intentional?

Reply:
Not intentional, for sure.